test/sqllogictest/transform/column_knowledge.slt

```diff
@@ -774,22 +774,22 @@ WITH MUTUALLY RECURSIVE
 SELECT * FROM c0;
 ----
 Explained Query:
-  Return // { arity: 3, types: "(integer, integer?, integer?)" }
-    Get l1 // { arity: 3, types: "(integer, integer?, integer?)" }
+  Return // { arity: 3, types: "(integer, integer, integer?)" }
+    Get l1 // { arity: 3, types: "(integer, integer, integer?)" }
```

Comment:
Spot-check example: `t1.f1` is marked `PRIMARY KEY` and `t1.f2` is not, and so only the `f2` column ends up nullable.